src/types/Sort.ts

```diff
@@ -0,0 +1,5 @@
+export enum Sort {
+  all = 'all',
+  completed = 'completed',
+  active = 'active',
+}
```

Comment:
```suggestion
export enum Sort {
  All = 'all',
  Completed = 'completed',
  Active = 'active',
}
```
Enum keys should be capitalized